smoke-tests/matrix/src/main/docker/payara/launch.bat

```diff
@@ -1,2 +1 @@
-java -jar glassfish/lib/client/appserver-cli.jar start-domain domain1
-powershell -command "Get-Content /server/glassfish/domains/domain1/logs/server.log -Wait"
+java %JVM_ARGS% -jar glassfish/lib/client/appserver-cli.jar start-domain --verbose domain1
```

Comment:
`--verbose`?

Reply:
`--verbose` basically means start in foreground, pretty much the same as `catalina.sh run`